migrations/v2.js

```diff
@@ -0,0 +1,225 @@
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _canShowFeedback has been modified to false');
+    return true;
+  });
+
+  /**
+   * * Modify existing attribute to new value.
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._canShowMarking', async () => {
+    assessments.forEach(assessment => {
+      if (assessment._canShowMarking !== false) assessment._canShowMarking = false;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._canShowMarking attribute', async () => {
+    const isValid = assessments.every(assessment =>
+      assessment._canShowMarking === false
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _canShowMarking has been modified to false');
+    return true;
+  });
+
+  /**
+   * * Modify existing attribute to new value.
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._canShowModelAnswer', async () => {
+    assessments.forEach(assessment => {
+      if (assessment._canShowModelAnswer !== false) assessment._canShowModelAnswer = false;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._canShowModelAnswer attribute', async () => {
+    const isValid = assessments.every(assessment =>
+      assessment._canShowModelAnswer === false
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _canShowModelAnswer has been modified to false');
+    return true;
+  });
+
+  /**
+   * * Remove existing attribute.
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._requireAssessmentPassed', async () => {
+    assessments.forEach(assessment => {
+      delete assessment._requireAssessmentPassed;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._requireAssessmentPassed attribute', async () => {
+    const isValid = assessments.every(assessment =>
+      assessment._requireAssessmentPassed === undefined
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _requireAssessmentPassed has been removed');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-assessment - update to v2.1.1', { name: 'adapt-contrib-assessment', version: '2.1.1', framework: '>=2.0.18' });
+});
+
+describe('adapt-contrib-assessment - v2.1.1 > v2.2.0', async () => {
+  let articles, assessments;
+
+  whereFromPlugin('adapt-contrib-assessment - from v2.1.1', { name: 'adapt-contrib-assessment', version: '<2.2.0' });
+
+  whereContent('adapt-contrib-assessment - where assessment', async content => {
+    articles = content.filter(({ _type }) => _type === 'article');
+    assessments = articles.filter(({ _type, _assessment }) => _type === 'article' && _assessment !== undefined);
+    if (assessments) return true;
+  });
+
+  /**
+   * * Modify Attribute
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._includeInTotalScore attribute', async () => {
```

Comment:
`_includeInTotalScore` is an existing property and I think we're avoiding changing boolean defaults.

Reply:
Gotcha. Will remove.